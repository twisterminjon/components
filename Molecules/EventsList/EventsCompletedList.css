.eventscompletedlist-container {
  width: 100%;
}
