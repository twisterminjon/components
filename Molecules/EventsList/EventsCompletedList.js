import React from 'react'
import PropTypes from 'prop-types'

import EventCompletedCard from '../../Molecules/EventCard/EventCompletedCard'

EventsCompletedList.propTypes = {
  /** Array of events to show */
  events: PropTypes.arrayOf(PropTypes.object).isRequired,
}

export default function EventsCompletedList({ events }) {
  return (
    <div className="eventscompletedlist-container">
      {events.map(event => (
        <EventCompletedCard event={event} key={event.id} />
      ))}
    </div>
  )
}
