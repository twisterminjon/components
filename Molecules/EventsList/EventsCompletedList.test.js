import React from 'react'
import EventsCompletedList from './EventsCompletedList'

describe('EventsCompletedList', () => {
  it('matches the snapshot', () => {
    const wrapper = window.mount(<EventsCompletedList events={[]} />)

    expect(wrapper.find(EventsCompletedList)).toMatchSnapshot()
  })
})
