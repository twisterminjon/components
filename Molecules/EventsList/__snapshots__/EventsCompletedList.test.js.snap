// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventsCompletedList matches the snapshot 1`] = `
<EventsCompletedList
  events={Array []}
>
  <div
    className="eventscompletedlist-container"
  />
</EventsCompletedList>
`;
